lib/main.js

```diff
@@ -13,8 +13,10 @@ function _createWindow (options) {
     show: false
   }, options)
 
-  opts.preload = path.join(__dirname, 'renderer-preload')
-  if (v[0] === 0 && v[1] >= 37) {
+  // options.preload was deprecated for options.webPreferences.preload in 0.37.3
+  if (v[0] === 0 && (v[1] <= 36 || (v[1] === 37 && v[2] < 3))) {
```

Comment:
@jprichardson What are you thoughts on this conditional? Each piece could be its own variable if you think that's clearer.